### PR TITLE
Update installation dataloading docs

### PR DESCRIPTION
**Pull request by alexgleith**
### Reason for this pull request

Update, upgrade, move forwards and generally improve docs.

### Proposed changes

- Significant is de-emphasising ingestion
- Also replaced the indexing from S3 with something much simpler that still works.

 - [ ] Closes #xxxx
 - [ ] Tests added / passed
 - [ ] Fully documented, including `docs/about/whats_new.rst` for all changes

### CODE REVIEW

**Review by robbibt — APPROVED** [review]
This looks like an amazing set of changes to me - this section alone should reduce Slack questions by at least half! 
image

File: docs/ops/indexing.rst [hunk 1]
```diff
@@ -167,160 +182,76 @@ Everything is now ready, and we can use the **datacube** tool to add one or more
...
-__ https://lpdaac.usgs.gov/products/mcd43a4v006/
-
-Once you have downloaded some data, it will need :ref:`metadata preparation
-<prepare-scripts>` before use in the Data Cube.
-
 
 Indexing Data on Amazon(AWS S3)
 ===============================
 
+<<<<<<< HEAD
```
Comment (uchchwhash) [inline docs/ops/indexing.rst]: hmm... we need to address this conflict here

**Review by uchchwhash — APPROVED** [review]
All good changes, completely agree with @robbibt.

Please address the git conflict before merging though!

**Comment by RichardScottOZ** [comment]
also

'If you'd like your project to be on this list, please raise an issue or creat a pull request'

need an extra e in create in docs/user/deployments.rst  line 40 or so I think it said.

File: docs/ops/indexing.rst (hunk 1 above)
Comment (alexgleith) [reply docs/ops/indexing.rst]: Oops, missed one!

**Pull request merged**